src/blockly/generators/propc/base.js

```diff
@@ -1288,8 +1287,11 @@ Blockly.Blocks.comment = {
 
 Blockly.propc.comment = function () {
     var text = this.getFieldValue("COMMENT_TEXT");
-
-    return '// ' + text.replace(/,+\s*$/g, '') + '\n';
+    var code = '';
```

Comment:
It looks like this code is being fail-safe. Consider replacing
```
    var code = '';
    if (text) {
        code = '// ' + text.replace(/,+\s*$/g, '') + '\n';
    }
    return code;
```
with this:
```
return (text) ? '// ' + text.replace(/,+\s*$/g, '') + '\n' : '';
```
It eliminates a variable and is still unambiguous.

Reply:
@MatzElectronics reminder to do this.